### ISSUE TITLE
Empty block ░  is messed up

### ISSUE THREAD

**Issue by Mitrajit** [anmol098/waka-readme-stats#211]
From the very beginning, I found that the empty block ░ for bar graphs messes with the output.
<p>
<img src="https://user-images.githubusercontent.com/55190574/126076233-ed6180d7-be7b-416c-ade0-c434baf709d9.png" />
<img src="https://user-images.githubusercontent.com/55190574/126076442-c16bc6ec-b37f-4c01-9b67-43fefc268ead.png" alt="error" />
</p>
I have tried it on my friends' computer, different browser but it shows a similar problem. Even your readme preview have such messed up output. However, I get the expected output when I open it in Chrome for Android.
<br><br>

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to my[ GitHub profile](https://github.com/Mitrajit) readme 

**Expected behaviour**
Screenshot taken from my android device
<img src="https://user-images.githubusercontent.com/55190574/126076891-d78bc9e3-12c8-4ba3-8834-fb0f51ad7835.jpeg" width="400" />

**Github repository link**
https://github.com/Mitrajit

This might be a resolution related problem and hopefully, you will come up with a solution.
Thank you.

**Comment by anmol098** [comment]
For me, it's fine on your profile. there is some issue with the different resolution. I am using  Google Chrome | 91.0.4472.114 (Official Build) (x86_64)


<img width="874" alt="Screenshot 2021-07-21 at 4 31 18 PM" src="https://user-images.githubusercontent.com/15426564/126478549-8f116791-aded-4305-9e80-c4141908a7a0.png">

**Comment by aravindvnair99** [comment]
@Mitrajit I do have that issue on Windows for all resolutions except `2560 * 1080`. Which OS are you on?

@anmol098 Is that screenshot from macOS?

**Comment by anmol098** [comment]
> @Mitrajit I do have that issue on Windows for all resolutions except `2560 * 1080`. Which OS are you on?
> 
> @anmol098 Is that screenshot from macOS?

Yes I am on macOS

**Comment by Mitrajit** [comment]
@aravindvnair99 I am using Windows 10.
@anmol098 I think there should be some workaround for Windows machines with lower resolution screens especially because most laptops don't have that high resolution.

**Comment by aravindvnair99** [comment]
@anmol098 and @Mitrajit I experimented a bit and realised the problem is that the height isn't matching between ░ and ▊ thus making the former bigger as shown here: ░ ▊ Every OS uses their own font settings (style, width, height. thickness, etc)

If we can find a character that looks same on all OS and of same height, then the issue can be rectified.

**Comment by Mitrajit** [comment]
@anmol098 and @aravindvnair99 I was searching for a character that will suit all OS and look good at the same time. After looking into all the weird and complex Unicode characters I found `/` backslash U+002F is the easiest to code by just replacing the empty_block with `/`. Other characters like `⣿` have a different width than `█` that might cause alignment issues.
**Example**
```
💬 Programming Languages: 
C                        2 hrs 15 mins       ███████████//////////////   49.88% 
Java                     1 hr 6 mins         █████////////////////////   20.13% 
C++                      39 mins             ███//////////////////////   12.0% 
Python                   38 mins             ███//////////////////////   11.67% 
Markdown                 30 mins             /////////////////////////   0.32%

🔥 Editors: 
VS Code                  5 hrs 31 mins       █████████████████████████   100.0%
```
**Before and after screenshots**
<p>
<img src="https://user-images.githubusercontent.com/55190574/126508108-4c2b94eb-db23-48ca-ac1a-620d5a556876.png" height="125px"/>&nbsp;
<img src="https://user-images.githubusercontent.com/55190574/126508005-66329695-0f99-4a91-9d1d-cb059db8e867.png" height="125px"/>
</p>
Please say if I can give a PR for the same.

**Comment by aravindvnair99** [comment]
@Mitrajit That's great to hear! If you make a PR, I could test your suggestion.

**Comment by Mitrajit** [comment]
@aravindvnair99 Unfortunately I just realised `/` is having a different width for chrome in android. Can you please open this issues page on your mobile device and check the progress bars?
Mine is terribly misaligned.
<img src="https://user-images.githubusercontent.com/55190574/126513071-965d6517-be61-4de0-a4ac-79f9587262ae.jpg" width="600px"/>
We might need to find some other solution.

**Comment by aravindvnair99** [comment]
@Mitrajit Good findings! I agree, on my Windows laptop, it looks proper for all resolutions I tested; but not on my Android phone.

Seems like we need something that is the same for all OS as when I tried phone and tablet resolutions on my laptop, it comes correct. But not when I check on Android. So, as I mentioned in https://github.com/anmol098/waka-readme-stats/issues/211#issuecomment-884190440, the font is the issue as far as I understood as different OS use different fonts + the ability for users to change it as well from the defaults to something they prefer. I see two options here:

- Finding a character that looks the same on all platform's default fonts. (Still, there's a possibility of things going wrong on custom fonts)
- Generating an image and putting it in README instead of the characters

**Comment by Mitrajit** [comment]
@aravindvnair99 I found a work around that must work on all devices and platforms - *Tables*
<table>
<tr><th colspan="3">💬 Programming Languages: </th></tr>
  <tr>
    <td> C </td>
    <td> 2 hrs 15 mins </td>
    <td> ██████████░░░░░░░░░░░░░░░   40.88%  </td>
  </tr>
  <tr>
    <td> Java </td>
    <td> 1 hr 6 mins </td>
    <td> █████░░░░░░░░░░░░░░░░░░░░   20.13%  </td>
  </tr>
    <td>C++ </td>
    <td>39 mins </td>
    <td> ███░░░░░░░░░░░░░░░░░░░░░░   12.0%  </td>
  </tr>
</table>
<table>
<tr><th colspan="3">🔥 Editors: </th></tr>
  <tr>
    <td> VS Code </td>
    <td> 5 hrs 31 mins </td>
    <td>  █████████████████████████   100.0%</td>
  </tr>
</table>

> - Generating an image and putting it in README instead of the characters

I agree but images will not always match the theme (light/dark/dark dimmed) and if we want to use images, we can make things fancy with piecharts and other plotting models.

**Comment by aravindvnair99** [comment]
@Mitrajit That's great! You could make a PR. The table looks good to me on Windows, iOS, and Android devices. I don't have my Mac with me at the moment to test. @anmol098 How does it look for you?

**Comment by Koushikphy** [comment]
have this same issue on chrome in windows 10

**Comment by aravindvnair99** [comment]
Fixed in #266. Feel free to create a new ticket for any further issues.
